asteroidsyncserviced/bluez/bluezclient.h

```diff
@@ -57,11 +56,7 @@ private slots:
 
 private:
     bool isAsteroidOSWatch(const QStringList uuids) const;
-    QDBusConnection m_dbus;
-    DBusObjectManagerInterface m_bluezManager;
-    BluezAdapter1 *m_bluezAdapter = nullptr;
-    FreeDesktopProperties *m_bluezAdapterProperties = nullptr;
-
+    BluezClientImpl *bt = nullptr;
```

Comment:
I can't make sense of "bt" ? If you follow Qt practices this would be something like d_ptr.

Reply:
You're right.  Done.